hedera-node/configuration/mainnet/application.properties

```diff
@@ -1,3 +1,6 @@
+# This is exact replica of file 121 on mainnet and must match bootstrap.properties exactly.
+# It will be used by modular code for all the property overrides.
+# It does not have to match bootstrap.properties once that file is deprecated by the switch from mono-service to hedera-config.
```

Comment:
```suggestion
# This is an exact replica of file 121 on mainnet and must match bootstrap.properties exactly.
# It will be used by modular code for all the property overrides.
# It does not have to match bootstrap.properties once that file is deprecated by the switch from using mono-service config to hedera-config.
```

Reply:
Fixed. Thanks!

Reply:
I think this one got missed.